fv3core/fv3core/_config.py

```diff
@@ -553,41 +597,47 @@ class Namelist:
     vtdm4: float = DEFAULT_FLOAT
     # warm_start: bool
     z_tracer: bool = DEFAULT_BOOL
-    # c_cracw: Any
-    # c_paut: Any
-    # c_pgacs: Any
-    # c_psaci: Any
-    # ccn_l: Any
-    # ccn_o: Any
-    # const_vg: bool
-    # const_vi: bool
-    # const_vr: bool
-    # const_vs: bool
-    # de_ice: Any
-    do_qa: bool = DEFAULT_BOOL
-    # do_sedi_heat: Any
-    # do_sedi_w: Any
-    # fast_sat_adj: bool
-    # fix_negative: bool
-    # irain_f: Any
-    # mono_prof: Any
-    # mp_time: Any
-    # prog_ccn: Any
-    # qi0_crt: Any
-    # qs0_crt: Any
-    # rh_inc: Any
-    # rh_inr: Any
+    c_cracw: float = NamelistDefaults.c_cracw
+    c_paut: float = NamelistDefaults.c_paut
+    c_pgacs: float = NamelistDefaults.c_pgacs
+    c_psaci: float = NamelistDefaults.c_psaci
+    ccn_l: float = NamelistDefaults.ccn_l
+    ccn_o: float = NamelistDefaults.ccn_o
+    const_vg: bool = NamelistDefaults.const_vg
+    const_vi: bool = NamelistDefaults.const_vi
+    const_vr: bool = NamelistDefaults.const_vr
+    vs_fac: float = NamelistDefaults.vs_fac
+    vg_fac: float = NamelistDefaults.vg_fac
+    vi_fac: float = NamelistDefaults.vi_fac
+    vr_fac: float = NamelistDefaults.vr_fac
+    de_ice: bool = NamelistDefaults.de_ice
+    do_qa: bool = NamelistDefaults.do_qa
+    do_sedi_heat: bool = NamelistDefaults.do_sedi_heat
+    do_sedi_w: bool = NamelistDefaults.do_sedi_w
+    fast_sat_adj: bool = NamelistDefaults.fast_sat_adj
+    fix_negative: bool = NamelistDefaults.fix_negative
+    irain_f: float = NamelistDefaults.irain_f
```

Comment:
This one should be a boolean too

Reply:
that one is set to a integer in fortran (though the name sounds like it should be a float haha)